knimin/lib/tests/test_data_access.py

```diff
@@ -297,6 +297,99 @@ def test_get_ag_barcode_details(self):
             del(obs[key]['registered_on'])
         self.assertEqual(obs, exp)
 
+    def test_plate_operations(self):
```

Comment:
Nope, that is ok. If there is a bug in any of them, the specific test will fail, so we can easily identify which one is the source of the problem.